packages/components/src/date-time/date/styles.ts

```diff
@@ -35,6 +39,7 @@ export const Calendar = styled.div`
 
 export const DayOfWeek = styled.div`
 	color: ${ COLORS.gray[ 700 ] };
+	font-size: ${ CONFIG.fontSize };
```

Comment:
We also need to set the `line-height`, otherwise it will get overridden by WordPress styles

```suggestion
	font-size: ${ CONFIG.fontSize };
	line-height: ${ CONFIG.fontLineHeightBase };
```